swift-sdk/Internal/InAppHelper.swift

```diff
@@ -472,8 +474,9 @@ struct InAppHelper {
     
     private enum UrlScheme : String {
         case applewebdata = "applewebdata"
-        case itbl = "itbl"
+        case iterable = "iterable"
         case action = "action"
+        case backwardCompat = "itbl"
```

Comment:
I think it might be a bit clearer to name this itbl and add a comment that it's for backwards compatibility since all the other values are following the convention of using the scheme name itself.

Reply:
Will fix this.